src/main/java/org/apache/skywalking/banyandb/v1/client/MeasureQuery.java

```diff
@@ -89,6 +98,18 @@ public MeasureQuery sumBy(String field, Set<String> groupByKeys) {
         return this;
     }
 
+    public MeasureQuery topN(int number, String field) {
+        Preconditions.checkArgument(fieldProjections.contains(field), "field should be selected first");
+        this.topN = new TopN(field, number, Sort.DESC);
+        return this;
+    }
+
+    public MeasureQuery bottomN(int number, String field) {
+        Preconditions.checkArgument(fieldProjections.contains(field), "field should be selected first");
+        this.topN = new TopN(field, number, Sort.ASC);
+        return this;
+    }
```

Comment:
How about using one function? `limit(int number, String field, Sort order)`?

Reply:
Do we need to update readme.md?

Reply:
Notice, `Sort` is a gRPC level object, we need a wrapper.

Reply:
> Notice, `Sort` is a gRPC level object, we need a wrapper.

`Sort` is already a wrapper here. It exists in the AbstractQuery class as an Enum.

Reply:
> Do we need to update readme.md?

Yes. I think so, we have to add more use cases.

Reply:
docs updated

Reply:
I prefer to still use `bottomN` and `topN` in order to distinguish it from `limit` and `offset`.